docs/guide/actions.md

```diff
@@ -32,25 +32,32 @@ You will not typically need this in a real app as Textual will run actions in li
 
 ## Syntax
 
+Action handling can only take place in three objects:
+
+  - the widget containing the renderable of which the action string is a part,
+  - the Screen containing that widget, or
+  - the App
+
 Action strings have a simple syntax, which for the most part replicates Python's function call syntax.
 
 !!! important
 
-    As much as they *look* like Python code, Textual does **not** call Python's `eval` function to compile action strings.
+    Though they *look* like Python code, Textual does **not** call Python's `eval` function to compile action strings.
+
+You can specify handling in either an app or a screen action method by starting the action string with a namespace (either `"screen"` or `"app"` followed by a dot). Otherwise the appropriate action method is sought first in the widget, then in the screen, and finally in the app. It's important to note the difference between this and the bubbling sequence used to process events.
 
-Action strings have the following format:
+Action strings therefore have the following format:
 
-- The name of an action on is own will call the action method with no parameters. For example, an action string of `"bell"` will call `action_bell()`.
-- Actions may be followed by braces containing Python objects. For example, the action string `set_background("red")` will call `action_set_background("red")`.
-- Actions may be prefixed with a _namespace_ (see below) followed by a dot.
+- The name of an action (with or without a namespace) on its own will call the action method with no parameters in the given namespace, if any. For example, an action string of `"app.bell"` will call your app's `action_bell()` method.
+- The action name may be followed by parentheses optionally containing a list of Python objects, which become parameters to the action method call. For example, the action string `set_background("red")` will first try to call `action_set_background("red")` as a widget method, then as a screen method, and finally as an app method.
```

Comment:
This is simply a refactoring of the explanation. The first bullet explains parameterless actions while pointing out this is orthogonal to the presence of a namespace. The second demonstrates actions with parameters and explains the lookup sequence when no namespace is present. "Braces" was, of course, just plain wrong, so I presume it's a holdover from an earlier design.